src/transformState.js

```diff
@@ -4,8 +4,33 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const objects of actions) {
+    const { type, extraData, keysToRemove } = objects;
+
+    switch (type) {
+      case 'addProperties' :
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties' :
+        for (const deleteValue of keysToRemove) {
```

Comment:
This would make your code clearer
```suggestion
        for (const keyToRemove of keysToRemove) {
```